cxx-squid/src/main/java/org/sonar/cxx/preprocessor/CxxPreprocessor.java

```diff
@@ -743,19 +743,20 @@ private int matchArguments(List<Token> tokens, List<Token> arguments) {
           break;
         }
       } while (true);
-
+    } catch (MismatchException me) {
+      LOG.debug("Mismatch: expected ','got: '" + rest.get(0).getValue() + "'");
```

Comment:
Typically this message is: 

> Mismatch: expected ',' got: ')' 

This is the reason for the exception but not an error. Anything different than ',' or ')' is an error. This message could be an error if the ')' case is not reported.